.kokoro/Dockerfile

```diff
@@ -47,7 +47,7 @@ ARG SCCACHE="$(which sccache)"
 
 WORKDIR /pytorch/xla
 ARG GCLOUD_SERVICE_KEY_FILE="/pytorch/xla/default_credentials.json"
-ARG SILO_NAME='cache-silo-ci-gcc-11'  # cache bucket for CI
+ARG SILO_NAME='cache-silo-ci-dev-container-python38-bullseye'  # cache bucket for CI
```

Comment:
https://github.com/pytorch/xla/pull/5290/files#r1339193970

Reply:
sg, will update the name here